cryptopan/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "cryptopan"
+version = "0.1.0-pre"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+description = "CryptoPAN algorithm for IP address anonymization"
+repository = "https://github.com/RustCrypto/utils"
+keywords = ["cryptopan", "ip anonymization"]
+categories = ["cryptography"]
+edition = "2021"
+readme = "README.md"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+aes="0.8.1"
+bitvec = "1.0.1"
+ecb="0.1.1"
```

Comment:
You don't need the `ecb` crate anymore. `BlockEncrypt`/`BlockDecrypt` trait impls on cipher types can be used to perform ECB. Also please add trailing newline.